### PR TITLE
[IMP] mail: introduce RtcController/microphoneButtonTitle

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/models/rtc_controller.js [hunk 1]
```diff
@@ -80,6 +80,17 @@ registerModel({
         _computeIsSmall() {
             return Boolean(this.callViewer && this.callViewer.threadView.compact && !this.callViewer.isFullScreen);
         },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeMicrophoneTitle() {
+            if (this.messaging.rtc.currentRtcSession.isMute) {
```
Comment (oomsveta) [inline addons/mail/static/src/models/rtc_controller.js:88]: You should ensure that `currentRtcSession` is defined before trying to read `isMute`.

**Review by oomsveta — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/models/rtc_controller.js (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/models/rtc_controller.js:88]: Same as in the `headphoneTitle`, it wasn't guarded in template, so I assume it never happens to have `RtcController` without a `currentRtcSession`

File: addons/mail/static/src/models/rtc_controller.js [hunk 2]
```diff
@@ -28,6 +28,9 @@ registerModel({
          * @param {MouseEvent} ev
          */
         onClickMicrophone(ev) {
+            if (!this.messaging.rtc.currentRtcSession) {
+                return clear();
+            }
```
[the remark this answers is not in the document]
Comment (alexkuhn) [reply addons/mail/static/src/models/rtc_controller.js]: Oops, change of erroneous "change done" done

File: addons/mail/static/src/models/rtc_controller.js [hunk 3]
```diff
@@ -80,6 +80,20 @@ registerModel({
         _computeIsSmall() {
             return Boolean(this.callViewer && this.callViewer.threadView.compact && !this.callViewer.isFullScreen);
         },
+        /**
+         * @private
+         * @returns {string}
```
Comment (oomsveta) [inline addons/mail/static/src/models/rtc_controller.js:85]: string or FieldCommand

**Review by oomsveta — APPROVED** [review]

**Pull request closed without merging**